### DISCUSSION TITLE
请问PAG支持某个图层动态添加或替换成UIView或者CALayer吗？

### DISCUSSION THREAD

**Discussion by Bruce-pac** [Tencent/libpag#1930] [Q&A] [answered]
类似于SVGA的动态绘制一样，
`[aPlayer setDrawingBlock:^(CALayer *contentLayer, NSInteger frameIndex) {
                [contentLayer addSublayer:otherView.layer];
} forKey:@"banner"];`

**Comment by kevingpqi123** [comment, marked as answer]
其实你应该简述你要实现的是什么功能，可能最优的方案不是你现在所想的

pag 对齐的是 AE，还原的是 AE 的效果，pag 可以作为模版使用，支持编辑的只能是内部的渲染树，不能添加系统 View
